core/templates/pages/exploration-editor-page/exploration-editor-page.module.ts

```diff
@@ -21,8 +21,11 @@ import 'zone.js';
 
 import 'angular-ui-sortable';
 import uiValidate from 'angular-ui-validate';
+import 'third-party-imports/guppy.import';
+import 'third-party-imports/midi-js.import';
 import 'third-party-imports/ng-audio.import';
 import 'third-party-imports/ng-joy-ride.import';
+import 'third-party-imports/skulpt.import';
```

Comment:
Could any of these newly added libs be only imported at places where they are actually needed? Like in some directive where they are used.

Reply:
Actually the issue with that is these imports look like this

`window.Guppy = ...`

and requiring this multiple times loads the library script many times. Plus anyways they need to be loaded in exploration editor and player page. So, added these here to load the script only once.